scripts/dpla_ingest_update.py

```
@@ -0,0 +1,102 @@
+WP_PASSWORD = os.environ['DPLA_WP_APP_PASSWORD']
+DPLA_KEY    = os.environ['DPLA_API_KEY']
+SLACK_URL   = os.environ['DPLA_SLACK_WEBHOOK']
+
+auth    = (WP_USER, WP_PASSWORD)
+headers = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36'}
+now       = datetime.now()
+yesterday = (now - timedelta(days=1)).strftime('%Y-%m-%d')
+
+# --- Fetch DPLA data ---
+providerurl  = f'https://api.dp.la/v2/items?facets=provider.name&api_key={DPLA_KEY}&page_size=0&facet_size=100'
+providernames = json.loads(requests.get(providerurl).text)['facets']['provider.name']['terms']
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)